server/mentions.go

```diff
@@ -241,10 +244,10 @@ func (p *NotificationsParser) sendChannelNotification(un *UserNotification, onli
 }
 
 func (p *NotificationsParser) sendUserActivity(userActivity *UserActivity) error {
-	sender, err := p.PAPI.GetUser(userActivity.UserNotification.Post.UserId)
-	if err != nil {
-		p.PAPI.LogError("Failed to get sender", "error", err.Error())
-		return err
+	sender, appErr := p.PAPI.GetUser(userActivity.UserNotification.Post.UserId)
```

Comment:
nit: can use the plugin api wrapper exposed on `p` to get real `error` values and avoid the `appErr` juggling.